tests/components/DBTableMockerTest.php

```diff
@@ -15,17 +15,39 @@ class DBTableMockerTest extends TestCase {
 
         /**
          * @covers ::getTableDescription
+         * @covers ::getCurrentColumn
          * 
          * @dataProvider provideTableNamesAndPrimaryKeys
          *
-         * @param string $tableName - name of the described table
+         * @param string $tableName         - name of the described table
          * @param (string|null)[] $expected - expected result
          * @return void
          */
         public function testGetTableDescription(string $tableName, array $expected):void {
             $tableMocker = new DBTableMocker();
             
             $this->assertContains($expected, $tableMocker->getTableDescription($tableName));
+            $this->assertEquals("", $tableMocker->getCurrentColumn());
+        }
+
+        /**
+         * @covers ::getTableDescription
+         * @covers ::column
+         * @covers ::getCurrentColumn
+         * 
+         * @dataProvider provideColumns
+         *
+         * @param string $table  - table which the column belongs t0
+         * @param string $column - name of the column
+         * @return void
+         */
+        public function testColumnMethodSetsUpWorkingColumn(string $table, string $column):void {
+            $tableMocker = new DBTableMocker();
+
+            $tableMocker->getTableDescription($table);
+
+            $this->assertNull($tableMocker->column($column));
+            $this->assertEquals($column, $tableMocker->getCurrentColumn());
```

Comment:
This test asserts that the given column is actually stored and can be worked with later.